test/discovery/HazelcastCloudDiscoveryTest.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var HazelcastClient = require("../../").Client;
+var expect = require('chai').expect;
+
+var ClientConfig = require("../../lib/config/Config").ClientConfig;
+
+describe('HazelcastCloudDiscovery Test', function () {
+
+    // add token and group name and activate the test
+    xit('basic cloud discovery test', function () {
+        var clientConfig = new ClientConfig();
+        clientConfig.networkConfig.cloudConfig.enabled = true;
+        clientConfig.networkConfig.cloudConfig.discoveryToken = "EXAMPLE_TOKEN";
+        clientConfig.groupConfig.name = "GROUP_NAME";
+        HazelcastClient.newHazelcastClient(clientConfig).then((client) => {
+            var mp = client.getMap("testMap");
```

Comment:
In a seperate PR, I think we need to enable code style checks for tests. Here we use single quotes.